### PR TITLE
New packages: Parsec

### PR DESCRIPTION
**Pull request by 3rav**

### CODE REVIEW

File: mingw-w64-parsec/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	 "${MINGW_PACKAGE_PREFIX}-ninja")
+makedepends=("flex"
+             "bison"
+             "${MINGW_PACKAGE_PREFIX}-hwloc"
+             "${MINGW_PACKAGE_PREFIX}-ninja"
+             "${MINGW_PACKAGE_PREFIX}-cmake")
+#optdepends=("")
+license=('custom')
+url='http://icl.utk.edu/parsec/'
+source=("${_realname}-${pkgver}"::"git+https://bitbucket.org/icldistcomp/${_realname}/src/master/")
```
Comment (lazka) [inline mingw-w64-parsec/PKGBUILD:21]: We don't allow packages that aren't pinned to a commit

File: mingw-w64-parsec/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+  MSYS2_ARG_CONV_EXCL="-DCMAKE_INSTALL_PREFIX=" \
+    ${MINGW_PREFIX}/bin/cmake \
+    -G'Ninja' \
+    -DCMAKE_INSTALL_PREFIX=${MINGW_PREFIX} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCUDA_USE_STATIC_CUDA_RUNTIME=OFF \
+    -DPARSEC_GPU_WITH_CUDA=OFF \
+    -DPARSEC_WITH_DEVEL_HEADERS=ON \
+    -DBUILD_SHARED_LIBS=OFF \
```
Comment (lazka) [inline mingw-w64-parsec/PKGBUILD:37]: Why no shared libaries?

Comment (3rav) [reply mingw-w64-parsec/PKGBUILD:37]: For this version I was unable to compile the dynamic version, however there is a fix now - after approval:
https://bitbucket.org/icldistcomp/parsec/pull-requests/454

Comment (lazka) [reply mingw-w64-parsec/PKGBUILD:37]: ok, thanks :+1:

File: mingw-w64-parsec/PKGBUILD (hunk 1 above)
Comment (3rav) [reply mingw-w64-parsec/PKGBUILD:21]: May I have a more extensive explanation?

Comment (lazka) [reply mingw-w64-parsec/PKGBUILD]: I've added a commit to this PR that shows what I mean: 2165037bf6680

**Comment by lazka** [comment]
@3rav are you OK with my changes?

**Comment by 3rav** [comment]
> @3rav are you OK with my changes?
OK :-)

**Pull request merged**